src/style.css

```diff
@@ -0,0 +1,161 @@
+/* #region Normalize */
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+/* #endregion */
+
+/* #region variables */
+:root {
+  --standart-margin: 16px;
+}
+/* #endregion */
+
+/* #region Card */
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img-container {
+  display: flex;
+  justify-content: center;
+  margin-top: 32px;
+  margin-bottom: 40px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+}
+
+.card__text-container {
+  display: flex;
+  margin-left: var(--standart-margin);
+  margin-bottom: 4px;
+}
+
+.card__text {
+  margin: 0;
+  line-height: 18px;
+  text-decoration: none;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  color: #060b35;
+}
+
+.card__product-code {
+  margin: 0;
+  margin-left: var(--standart-margin);
+  margin-bottom: var(--standart-margin);
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  color: #616070;
+}
+
+.card__feedback {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  text-align: center;
+  margin: 0 var(--standart-margin);
+  margin-bottom: 24px;
+}
+
+.card__feedback-text {
+  margin: 0;
+  line-height: 14px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  color: #060b35;
+}
+
+.card__price {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0 var(--standart-margin);
+}
+
+.card__price-text {
+  margin: 0;
+  line-height: 18px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+  color: #616070;
+}
+
+.card__price-value {
+  margin: 0;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  color: #060b35;
+}
+
+.card__buy-link {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  box-sizing: border-box;
+  height: 40px;
+  margin: var(--standart-margin);
+  text-decoration: none;
+  line-height: 16px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 14px;
+  border: 1px solid transparent;
+  border-radius: 5px;
+  color: white;
+  background-color: #00acdc;
+}
+
+.card__buy-link:hover {
+  background-color: white;
+  border: 1px solid #00acdc;
+  color: #00acdc;
+}
+/* #endregion */
+
+/* #region Stars */
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: flex;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--5 .stars__star,
```

Comment:
Slit your css code to different files - BEM blocks.

According to [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)

`11.[FILE STRUCTURE] - You should create separate file for each BEM block`

detailed info how to do it in [this video](https://mate.academy/learn/css-advanced/bem#/video/917)